src/derivative_operators/abstract_operator_functions.jl

```diff
@@ -209,3 +209,11 @@ function Base.sparse(A::AbstractDerivativeOperator{T}) where T
     end
     return mat
 end
+
+#=
+    Fallback methods that use the full representation of the operator
+=#
+Base.expm(A::AbstractDerivativeOperator{T}) where T = expm(full(A))
+Base.:/(A::AbstractVecOrMat, B::AbstractDerivativeOperator) = A / full(B)
+Base.:/(A::AbstractDerivativeOperator, B::AbstractVecOrMat) = full(A) / B
+# Base.:\ is also defined
```

Comment:
IIRC isn't `/` defined in terms of `\` with a transpose? It may be better to do that one directly. I would use `@which` to dive into the Base code and see how `/` is called.

Reply:
I commented out this portion and found that 
```julia
using DiffEqOperators
N = 20
A = DerivativeOperator{Float64}(2,2,1.0,N,:Dirichlet0,:Dirichlet0)
mat1 = rand(N,N)

@which A / mat1
no method found for the specified argument types

@which mat1' \ A'
\(x, y) at operators.jl:457
```

The definition is
```julia
\(x,y) = (y'/x')'
```

So Julia does in fact define `\` in terms of '/' for generic types. However for regular matrices the relationship is inverted:
```julia
# In linalg\generic.jl:820
(/)(A::AbstractVecOrMat, B::AbstractVecOrMat) = (B' \ A')'
```

Do you think I should define '/' using this convention, or just go with the generic one?

Reply:
I think you might need both. The transpose operator will be hard to do in general, and right now I think it's a no-op which only holds for some BCs and since we have the operator as `n x n`. Transpose on the dense matrix will always work though.